Configuration/QualificationTokens.json

```diff
@@ -779,44 +779,23 @@
     },
     {
         "accessToken": "Bearer 5d43166e-0d05-4ce5-8860-e4151ce44566",
-        "resourceId": "patient-van-de-XXX-Broekmeulen",
-        "familyName": "van de XXX_Broekmeulen",
-        "bsn": "999900134",
+        "resourceId": "patient-XXX-Drijkoningen",
+        "familyName": "XXX_Drijkoningen",
+        "bsn": "999900523",
         "qualificationScript": "CiO Test"
     },
     {
         "accessToken": "Bearer a00e10e0-0bee-4f0f-9974-1e92f02e6d92",
-        "resourceId": "patient-van-XXX-Lonkhuijsen",
-        "familyName": "van XXX_Lonkhuijsen",
-        "bsn": "999900171",
+        "resourceId": "patient-XXX-Dubbeld",
+        "familyName": "XXX_Dubbeld",
+        "bsn": "999900535",
         "qualificationScript": "CiO Test"
     },
     {
         "accessToken": "Bearer 2cabfc9e-6d89-4551-9183-6569699ce58f",
-        "resourceId": "patient-van-XXX-Verbeek",
```

Comment:
Klopt het dat deze patiënten verwijderd moeten worden?

Reply:
Ja; de gehele set van testmateriaal is herzien tussen beta.1 en beta.2, en er zijn ook (deels) andere Patiënten gebruikt. De oude Patiënten zijn niet meer nodig, omdat de bijbehorende testscripts ook niet meer gebruikt worden.